configuration-api-lib/src/commonMain/kotlin/co/electriccoin/zcash/configuration/model/map/Configuration.kt

```diff
@@ -0,0 +1,47 @@
+package co.electriccoin.zcash.configuration.model.map
+
+import co.electriccoin.zcash.configuration.model.entry.ConfigKey
+import kotlinx.datetime.Instant
+
+/**
+ * An immutable snapshot of a key-value configuration.
+ */
+interface Configuration {
+    /**
+     * @return When the configuration was updated. Null indicates the configuration either doesn't refresh or has never been refreshed.
+     */
+    val updatedAt: Instant?
```

Comment:
Does this need a special logic to compare between Instant in the context of a Configuration ?